helmfile/values/opensearch/master.yaml.gotmpl

```diff
@@ -1,3 +1,10 @@
+{{ if and .Values.opensearch.dataNode.dedicatedPods (eq .Values.opensearch.dataNode.count 0) }}
+{{ fail "\nERROR: Using 0 replicas with dedicated data pods is not supported by OpenSearch" }}
+{{ end }}
+{{ if and .Values.opensearch.clientNode.dedicatedPods (eq .Values.opensearch.clientNode.count 0) }}
+{{ fail "\nERROR: Using 0 replicas with dedicated client pods is not supported by OpenSearch" }}
+{{ end }}
```

Comment:
Move this to the release values specific to the data and client pods, and then just have the count in the condition.

Reply:
If I do that you can still apply only `app=opensearch-master` which will still get an invalid configuration.

Reply:
I guess that is true but then move it to `opensearch-common.yaml.gotmpl`.

Reply:
Done :+1: